dradis/dradis-ce
# Project show turbolinks

### Spec

Browsing back and forth Project#show in a project with findings and methodologies would some times result in a "black" findings chart (same column count and sizes, but all filled in black, that disappears after a second.

I think it may have to do with how we draw the SVG canvas for the issue / methodology charts.


**Proposed solution**

Review the JS code involved in Projects#show both in CE and Pro to make sure we're hooking into the right Turbolinks events.


### How to test

1. Load a Welcome project and open it.
2. Navigate outside of the summary and back into it to see the "black chart" glitch.
3. Go to Methodologies add some boards / lists / cards.
4. Navigate back to Projects#show to see the weirdness of the methodology chart.
